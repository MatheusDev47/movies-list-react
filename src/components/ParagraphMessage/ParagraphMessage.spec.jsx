import { render, screen } from '@testing-library/react';

import { ParagraphMessage } from './';

describe('<ParagraphMessage />', () => {
  it('Should render message in screen', () => {
    render(<ParagraphMessage message="Message test" />);

    expect(screen.getByText(/message test/i)).toBeInTheDocument();
  });

  it('Should match snapshot', () => {
    const { container } = render(<ParagraphMessage message="Message test" />);

    // eslint-disable-next-line testing-library/no-node-access
    expect(container.firstChild).toMatchSnapshot();
  });
});
