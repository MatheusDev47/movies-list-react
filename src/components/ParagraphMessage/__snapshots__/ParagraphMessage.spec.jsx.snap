// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ParagraphMessage /> Should match snapshot 1`] = `
<p
  class="p"
>
  Message test
</p>
`;
